packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -31,6 +32,7 @@ const WalletButton: React.FC<WalletButtonProps> = ({
     rounded = 'sm',
     size = 'md',
     text,
+    type,
```

Comment:
does it need to be prop or should it just be added to html permanently?

Reply:
@wojciech-deriv 
actually i faced with an issue when i needed to change the type of the button. so i think that we can leave it as a prop

Reply:
ah, okay, yes make sense then to have prop if you actually need to differentiate between various types of buttons,